packages/loot-core/src/types/api-handlers.d.ts

```diff
@@ -151,6 +151,11 @@ export interface ApiHandlers {
 
   'api/payee-delete': (arg: { id }) => Promise<unknown>;
 
+  'api/payees-merge': (arg: {
+    targetId: string;
+    mergeIds: string[];
+  }) => Promise<unknown>;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind adding a proper type here?
```suggestion
  }) => Promise<Something>;
```

Reply:
I'm not sure what type it returns, it seems to pass the return from the database back but when I log out the response from the API call it's always "undefined"

Reply:
If it's returning undefined then we can change it to `Promise<void>`

Reply:
Ah yes, just found it. Updated

https://github.com/actualbudget/actual/blob/6dd34b0c63b54c272c57063ceca7f43f39672df8/packages/loot-core/src/server/db/index.ts#L493-L528